src/cli/cert.rs

```diff
@@ -66,6 +82,27 @@ pub fn run(cert_args: CertArgs) -> exitcode::ExitCode {
                 println!("{}", serde_json::to_string(&success_msg).unwrap());
             };
         }
+        CertCommands::Upload(upload_args) => {
+            let api_key = get_api_key!();
```

Comment:
Could be nice to default the cert path to the value in the toml if not given

Reply:
Updated as well!